projects/adf-office-services-ext/src/lib/evaluators.spec.ts

```diff
@@ -268,6 +283,7 @@ describe('evaluators', () => {
 
     it('should return [false] if permissions check is false', () => {
       const context: any = {
+        aosPlugin: true,
```

Comment:
do we really need to add it everywhere?

Reply:
Yes, I think it makes sense in the context of each test. With the introduction of this new boolean, the remain tests will only make sense if the **aosPlugin** is true. If the **aosPlugin** is false, that means we won't have an AOS plugin to test all the other things.

Reply:
it should be working fine thanks to the default value the following methods return when the property is not found
this.appConfig.get<boolean>('plugins.aosPlugin', true);